src/voting-metadata-api/src/env.ts

```diff
@@ -0,0 +1,31 @@
+export const DEFAULT_REGION = 'us-west-1'
+export const DEFAULT_CACHE_BUCKET = 'nft-voting-tool-api-developer-cache'
+export const CACHE_BUCKET_NAME = process.env.CACHE_BUCKET_NAME || DEFAULT_CACHE_BUCKET
+export const AWS_REGION = process.env.AWS_REGION || DEFAULT_REGION
+export const IPFS_API_TOKEN = process.env.IPFS_API_TOKEN
```

Comment:
This won't work. We deliberately don't inject secrets into lambda environment since they appear in plain text. We have a convention that we inject `{SECRET_NAME}_ARN`, which is an ARN to SecretManager that is then read once on app startup and pushed into `process.env.{SECRET_NAME}`. In the case of this API, it was implemented slightly differently to use DI instead (via the AWSSecretsService) - so you might want to keep that.

Reply:
I couldn't find any secrets or `_ARN` env keys, assumed it was not implemented in this project. It's a good feature but want to limit the scope until we are in a better place with the project